src/components/GrantRequest/GrantRequestDueDiligenceSection.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useMemo, useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { isEqual, sumBy } from 'lodash'
```

Comment:
pls import each function separately from 'lodash/(whatever)', otherwise the whole lodash package is being imported